packages/web/src/pages/search-page-v2/RecentSearches.tsx

```diff
@@ -331,6 +331,8 @@ export const RecentSearches = () => {
       pv='xl'
       w='100%'
       css={{ maxWidth: '688px' }}
+      backgroundColor='white'
+      border='default'
```

Comment:
Huh. Does it not come with these by default?

Reply:
No, by default there is no border applied and the backgroundColor is "default". Not confusing at all 😂